### PR TITLE
Add download functionality

### PR DESCRIPTION
**Pull request by Luigi-Crisci**
This pull request adds a simple download button to retrieve the outstanding generated SYCL name, so that you can share your findings with your friends more easily
- Depends on [html2Canvas.js](https://github.com/niklasvh/html2canvas)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,4 +45,19 @@
...
 
+<script>
+    function downloadSyclName(){
+        const screenshotTarget = document.querySelector("#sycl-name");
+        html2canvas(screenshotTarget, {backgroundColor : null}).then((canvas) => {
+            const base64image = canvas.toDataURL("image/png");
+            // var a = document.createElement("a"); //Create <a>
+            // a.href = 'data:application/octet-stream;base64,' + base64image;
+            // a.download = "Image.png"; //File name Here
+            //a.click(); //Downloaded file
```
Comment (fknorr) [inline index.html]: What's with the commented-out stuff?

**Review by fknorr — APPROVED** [review]
This will get SYCL names to T-Shirts and Mugs worldwide!

File: index.html (hunk 1 above)
Comment (Luigi-Crisci) [reply index.html]: Things that I forgot to delete due to lack of sleep

Comment (Luigi-Crisci) [reply index.html]: Removed :))

**Pull request merged**